xdsl/utils/exceptions.py

```diff
@@ -46,12 +93,9 @@ def __init__(self,
                  span: 'Span',
                  msg: str,
                  history: 'BacktrackingHistory' | None = None):
-        preamble = ""
```

Comment:
Do we lose the history with that commit?

Reply:
This does not loose parse history! The history is generated using the `DeferredExceptionMessage` now